bin/infection

```diff
@@ -10,27 +10,7 @@ require __DIR__ . '/../app/bootstrap.php';
 use Infection\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
 
-function prepareArgv()
-{
-    $argv = $_SERVER['argv'];
-
-    $found = false;
-
-    while (next($argv)) {
-        $value = current($argv);
-        if (!$value || '-' !== $value[0]) {
-            $found = true;
-        }
-    }
-
-    if (!$found) {
-        array_splice($argv, 1, 0,'run');
-    }
-
-    return $argv;
-}
-
-$input = new ArgvInput(prepareArgv());
+$input = new ArgvInput(Application::prepareArgv($_SERVER['argv']));
```

Comment:
why do we need an `ArgvInput` at all? Can't we just leave it to the `Application::run()` to figure out which input to use and how to prepare it?

Reply:
It was that way before. I tried to keep most logic intact, since that’s beyond  the purpose of this PR

Reply:
$this->setDefaultCommand('run');

Reply:
Well, we can get rid of it with a single line of code, but then another issue appears. The issue can be fixed #294, but I more wary on why that's needed in the first place. I think it deserves a closer look in a separate PR, how do you think?

Reply:
OK never mind, I found the problem. Could you see if b02e5b4 makes any sense?